### DISCUSSION TITLE
Key Plate and RBG

### DISCUSSION THREAD

**Discussion by ekchom** [JellyTitan/Sofle-Pico#43] [Q&A] [answered]
This is first time I am trying something on build side. so please pardon if any question which doesn't make sense 
1) I need to carry the keyboard in travelling, so adding a keyplate ( and fixing switches on it ) would be better choice as I learn.  Is there any keyplate which can be 3d-printed ?
2) Is it compulsory to have RGB LED's, what if I don't want LED's at all ? It looks too flunky 
3) Is it compulsory to have rotary encoders? Can I leave then free, I just don't like the idea of having it.

**Comment by JellyTitan** [comment, marked as answer]
Thanks for trying the Sofle Pico!
1. This keyplate/case is 3d printable, it's basically the same as the acrylic case, but with better key retention than the acrylic: https://www.printables.com/model/760098-sofle-pico-casehttps://www.printables.com/model/760098-sofle-pico-case <br> for traveling, i'd probably recommend this case instead, because the edge of the case comes up to the bottom of the keycaps, so if you throw it in your bag, the keys are less likely to come un-socketed. https://www.printables.com/model/969590-sofle-pico-cloud-casehttps://www.printables.com/model/969590-sofle-pico-cloud-case
2. You can skip the RGB install entirely. There is no functional impact.
3.  You can leave out the rotary encoders entirely. 

Let me know if you have any other questions!
